renovate.json

```
@@ -1,6 +1,6 @@
 {
   "extends": ["github>shelfio/renovate-config-public"],
-  "labels": ["dependencies", "backend"],
+  "labels": ["frontend"],
```

```suggestion
  "labels": ["frontend", "dependencies"],
```
Lets have dependencies label too